cpp/test/CMakeLists.txt

```diff
@@ -30,7 +30,8 @@ set(TIMEOUT 30)
 
 foreach(URL ${URL_LIST})
     message(STATUS "Trying to download from ${URL}")
-    file(DOWNLOAD ${URL} "${CMAKE_BINARY_DIR}/googletest-release-1.12.1.zip" STATUS DOWNLOAD_STATUS TIMEOUT ${TIMEOUT})
+    file(DOWNLOAD ${
+    URL} "${CMAKE_BINARY_DIR}/googletest-release-1.12.1.zip" STATUS DOWNLOAD_STATUS TIMEOUT ${TIMEOUT})
```

Comment:
The CI failed caused by this change. Consider reverting it?

Reply:
OK
But I just use the clang-format in this file. Maybe there are some problems in .clang-format?